tools/log-collector/log_collector.go

```diff
@@ -171,29 +184,41 @@ func (l *LogCollector) CollectLogsAndDump() error {
 }
 
 // getApiGVResources returns list of resources for given group version
-func (l *LogCollector) getAPIGVResources(apiGroupVersion string) (gVResources []apiv1.APIResource) {
+func (l *LogCollector) getAPIGVResources(apiGroupVersion string) (gVResources []apiv1.APIResource, err error) {
 
 	var gVResourcesList *apiv1.APIResourceList
-	gVResourcesList, _ = l.disClient.ServerResourcesForGroupVersion(apiGroupVersion)
+	gVResourcesList, err = l.disClient.ServerResourcesForGroupVersion(apiGroupVersion)
+	if err != nil {
```

Comment:
No need to check this error type. Directly return error

Reply:
resolved